src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplTest.java

```diff
@@ -0,0 +1,107 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import edu.cornell.kfs.sys.service.impl.fixture.AwsSecretePojoBasic;
+
+class AwsSecretServiceImplTest {
+    private static final String KFS_INSTANCE_NAMESPACE = "kfs/local-dev/";
+    private static final String KFS_SHARED_NAMESPACE = "kfs/kfs-shared/";
+
+    private static final String SINGLE_STRING_SECRET_KEY_NAME = "unittest/singlestring";
+    private static final String SINGLE_DATE_SECRET_KEY_NAME = "unittest/singledate";
+    private static final String BASIC_POJO_SECRET_KEY_NAME = "unittest/pojo";
+    private static final String SINGLE_BOOLEAN_SECRET_KEY_NAME = "unittest/singleboolean";
+    
+    private static final String SINGLE_STRING_SECRET_VALUE = "Test Value";
+    private static final String BASIC_POJO_STATIC_STRING_VALUE = "do not change me";
+    private static final int BASIC_POJO_NUMBER_VALUE = 1;
+    
+    
+    private AwsSecretServiceImpl awsSecretServiceImpl;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        awsSecretServiceImpl = new AwsSecretServiceImpl();
+        awsSecretServiceImpl.setAwsRegion("us-east-1");
```

Comment:
I would suggest moving the region string to its own constant.

Reply:
OK